cluster/common.sh

```diff
@@ -58,16 +60,17 @@ function create-kubeconfig() {
     )
   fi
   local user_args=()
-  if [[ -z "${KUBE_USER:-}" || -z "${KUBE_PASSWORD:-}" ]]; then
-    user_args+=(
-     "--token=${KUBE_BEARER_TOKEN}"
-    )
-  else
+  if [[ ! -z "${KUBE_USER:-}" && ! -z "${KUBE_PASSWORD:-}" ]]; then
```

Comment:
@jlowdermilk - If I set --username, --password, and --token at the same time, does kubectl config set-credentials return an error?